clustered/client/src/main/java/org/ehcache/clustered/client/internal/SimpleClusterTierManagerClientEntity.java

```diff
@@ -40,12 +43,30 @@
  */
 public class SimpleClusterTierManagerClientEntity implements ClusterTierManagerClientEntity {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(SimpleClusterTierManagerClientEntity.class);
+
   private final EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint;
   private final LifeCycleMessageFactory messageFactory;
 
   public SimpleClusterTierManagerClientEntity(EntityClientEndpoint<EhcacheEntityMessage, EhcacheEntityResponse> endpoint) {
     this.endpoint = endpoint;
     this.messageFactory = new LifeCycleMessageFactory();
+    endpoint.setDelegate(new EndpointDelegate<EhcacheEntityResponse>() {
+      @Override
+      public void handleMessage(EhcacheEntityResponse messageFromServer) {
+
+      }
+
+      @Override
+      public byte[] createExtendedReconnectData() {
+        return new byte[0];
+      }
+
+      @Override
+      public void didDisconnectUnexpectedly() {
+        LOGGER.info("CacheManager got disconnected from server");
```

Comment:
Could we maybe get some identifying information here?  Maybe if we `toString()` the endpoint?  That should know the identity of the entity.

Reply:
Sadly endpoint does not override toString